source/beamline/VESPERS/VESPERSAddOnsCoordinator.cpp

```diff
@@ -1836,3 +1854,24 @@ void VESPERSAddOnsCoordinator::restoreAddOnsWireVAndNStatus()
 	if(!addOnsWireNStatusControl_->withinTolerance(finalStatus))
 		addOnsWireNStatusControl_->move(finalStatus);
 }
+
+void VESPERSAddOnsCoordinator::onMonoStatusChanged(bool isMoving)
+{
+	if (isMoving){
+
+		monoWaitTimer_->start(30000);
+		connect(monoFeedbackControl_, SIGNAL(valueChanged(double)), monoWaitTimer_, SLOT(start()));
```

Comment:
Question here. Should we connect to the `start()` signal or the `stop()` signal? I have thought that when Mono works as expected, it will always emit the valueChanged() signal.